### PR TITLE
Enhancement: Validation and Old Session

### PR DESCRIPTION
**Pull request by dayCod**
Changelog 23/7/2023 :
- Add UsersSeeder Class
- Update UsersFactory Class
- Add Old Session and Error Alert For Login Page
- Add Form Request Validation and Old Session For Create and Update Suppliers
- Add Form Request Validation and Old Session For Create and Update Customers 
- Add Form Request Validation and Old Session For Create and Update Unit
- Add Form Request Validation and Old Session For Create and Update Category
- Add Form Request Validation and Old Session For Create and Update Product

### CODE REVIEW

**Review by Bahrul-Rozak — COMMENTED** [review]
Thank you, i will review your changed later

File: database/seeders/UsersSeeder.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use App\Models\User;
+use Database\Factories\UserFactory;
+
+class UsersSeeder extends Seeder
```
Comment (Bahrul-Rozak) [inline database/seeders/UsersSeeder.php:10]: Good this is nice idea to adding factory seeder

File: app/Http/Controllers/Pos/CategoryController.php
Comment (Bahrul-Rozak) [inline app/Http/Controllers/Pos/CategoryController.php:1]: LGTM

File: app/Http/Controllers/Pos/CustomerController.php
Comment (Bahrul-Rozak) [inline app/Http/Controllers/Pos/CustomerController.php:1]: LGTM

File: app/Http/Controllers/Pos/ProductController.php
Comment (Bahrul-Rozak) [inline app/Http/Controllers/Pos/ProductController.php:1]: LGTM

File: app/Http/Controllers/Pos/SupplierController.php
Comment (Bahrul-Rozak) [inline app/Http/Controllers/Pos/SupplierController.php:1]: LGTM

File: app/Http/Controllers/Pos/UnitController.php
Comment (Bahrul-Rozak) [inline app/Http/Controllers/Pos/UnitController.php:1]: LGTM

**Review by Bahrul-Rozak — APPROVED** [review]
LGTM

File: database/seeders/UsersSeeder.php (hunk 1 above)
Comment (dayCod) [reply database/seeders/UsersSeeder.php:10]: yap more testable if your application need unit or feature test

**Pull request merged**